ts/screens/wallet/payment/TransactionErrorScreen.tsx

```diff
@@ -48,277 +70,289 @@ type Props = OwnProps &
   ReturnType<typeof mapStateToProps> &
   ReturnType<typeof mapDispatchToProps>;
 
-const styles = StyleSheet.create({
-  contentWrapper: {
-    alignItems: "center"
-  },
-
-  errorMessage: {
-    fontSize: customVariables.fontSize2,
-    paddingTop: customVariables.contentPadding,
-    textAlign: "center"
-  },
-
-  errorMessageSubtitle: {
-    textAlign: "center"
-  },
+const instabugTag = "payment-support";
```

Comment:
Since it is used twice, could be a refactoring place it in the util file `utils/payment` and share between the components?

Reply:
b3525ce to solve this